src/js/burials/config/form.js

```diff
@@ -160,7 +160,7 @@ const formConfig = {
             },
             veteranDateOfBirth: currentOrPastDateUI('Date of birth'),
             placeOfBirth: {
-              'ui:title': 'Place of birth'
+              'ui:title': 'Place of birth (city, state or country)'
```

Comment:
It looks like the text in other fields is `(city and state or foreign country)`, so let's use that.